CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -2,9 +2,9 @@ This document is a declaration of software quality for the `CycloneDDS` external
 
 # `CycloneDDS` Quality Declaration
 
-This quality declaration claims that `CycloneDDS` is in the **Quality Level 4** category.
+This quality declaration claims that `CycloneDDS` is in the **Quality Level 3** category.
 
-The `CycloneDDS` project meets a substantial number of the criteria stated for software quality.
+[The Eclipse Foundation](https://eclipse.org)'s `CycloneDDS` project meets a substantial number of the criteria stated for software quality.
 The developers have taken many steps to ensure that `CycloneDDS` is a software product which can be relied upon, however there is a significant lack of documentation regarding the interpretation of versions, features, and testing.
```

Comment:
I think you've shown that there isn't a significant documentation lack anymore and this sentence can be updated. Also, to update the sentence below, what do you feel is required in order to get to QL 2 or even QL 1?

Reply:
@brawner update, thanks! We need to go through the QL table again row by row to ascertain current level and what work remains.